### PR TITLE
Adds Base64 Data URI support

### PR DESCRIPTION
**Pull request by seanmetzgar**

### CODE REVIEW

**Comment by seanmetzgar** [comment]
Give this a whirl and make sure it's all working... I may have messed up the docs, not sure how to test that.

**Comment by seanmetzgar** [comment]
accept: text/plain : ?base64=true (svg data uri)
accept: text/plain : ?base64=true&format=svg (svg data uri)
accept: text/plain : ?base64=true&format=png (png data uri... it's BIG)

File: routes/faces/faces.go [hunk 1]
```diff
@@ -347,7 +353,17 @@ func GetFaceByValue(ctx *fiber.Ctx) error {
...
 	switch ctx.Get("accept") {
 	case "text/plain":
-		return ctx.SendString(graphic)
+		returnString := graphic
+
+		if ctx.Query("base64") == "true" {
+			if ctx.Query("format") == "png" {
+				returnString = "data:image/png;base64," + svg.SVGToPNGBase64(graphic)
+			} else {
+				returnString = "data:image/svg;base64," + b64.StdEncoding.EncodeToString([]byte(graphic))
```
Comment (GV14982) [inline routes/faces/faces.go:362]: Can you switch that prefix to `data:image:svg+xml;base64`?

I remember it causing a weird bug, but I don't remember the specifics.

**Review by GV14982 — CHANGES_REQUESTED** [review]

**Comment by GV14982** [comment]
I'll test this out later tonight or tomorrow.

**Comment by seanmetzgar** [comment]
I’m making some minor changes anyway… no worries

Sean Metzgar

Sent via Superhuman iOS ( ***@***.*** )

File: routes/faces/faces.go (hunk 1 above)
Comment (seanmetzgar) [reply routes/faces/faces.go]: good to go now 😸

**Comment by seanmetzgar** [comment]
cleaned this up a bit... just this now
accept: text/plain : ?base64=svg (svg data uri)
accept: text/plain : ?base64=png (png data uri... it's BIG)

**Review by GV14982 — APPROVED** [review]
Tested it out, nice job 👍🏻

**Pull request merged**